addOns/spiderAjax/src/main/resources/org/zaproxy/zap/extension/spiderAjax/resources/Messages.properties

```diff
@@ -1,8 +1,32 @@
+spiderajax.api.view.optionEventWait = Gets the event timer in miliseconds.
+spiderajax.api.view.optionMaxCrawlDepth = Gets the configured value for the scan crawl depth.
+spiderajax.api.view.optionMaxCrawlStates = Gets the configured value on how many crawl states it will show.
+spiderajax.api.view.optionMaxDuration = Gets the configured max duration of the crawl. The value is in minutes. 
+spiderajax.api.view.optionNumberOfBrowsers = Gets the configured number of browsers to be used.
+spiderajax.api.view.randominputs = Gets if the AjaxSpider will use random values in form fields when crawling, if set to true.
+spiderajax.api.view.optionReloadWait = Gets the configured time time to wait after reloading the page, this value is in miliseconds.
+spiderajax.api.view.results = Gets the current results of the crawler for the given site.
+spiderajax.pai.view.status = Gets the current status of the crawler. Actual values are Stopped and Running.
+spiderajax.api.action.scan = Runs the AjaxSpider against the given `URL`. URL parameter its mandatory and needs to have "http(s)://". `inScope` -> default its false, if you change this value to true, mean you already have this URL on your context/scope configuration. `ContextName` -> String value of any of your defined context you have, if the value here is not already defined, it will prompt an error. `subtreeOnly` -> default value is false, if changed to true it will crawl all the links on the web restricting the spider under the sites resources.
+spiderajax.api.action.scanAsUser = Runs the spider from the perspective of a User of the web aplication. The `userName` should be previusly defined on the context configuration. We recomend you set the url parameter, otherwise it's used an existing URL from the context (if any). The parameter `subtreeOnly` allows to restrict the spider under a site's subtree (using the specified 'url') default value here is false.
+spiderajax.api.action.setOptionBrowserId = Sets the configuration of the AjaxSpider to use one of the following suported browsers: Chrome, Chrome-headless, Firefox, Firefox-headless, HtmlUnit, PhantomJS, Safari. It can be changed at running time. Defaul is firefox-headless.
+spiderajax.api.action.setOptionClickDefaultElems = Setting values true or false, so the AjaxSpider will click on the default html tag values such as such as "a" "button" "input" that apears on the website. Defaul is true.
+spiderajax.api.action.setOptionClickElemsOnce = When enabled, the crawler attempts to interact with each element (e.g., by clicking) only once. If this is not set, the crawler will attempt to click multiple times. Unsetting this option is more rigorous but may take considerably more time. Sets the value to rue or false depending if we would like the AjaxSpider to click one time the items on the web. Default is true.
+spiderajax.api.action.setOptionEventWait = Sets the time to wait on each event in miliseconds. Default is 1000 miliseconds
```

Comment:
General question (cc @thc202 @kingthorin ) should we include default values in setters? I'm worried that this is another place to maintain this info, ie I can see the code and help getting out of date.

Reply:
That could be a good approach to maintain the documentation ap to date?

Reply:
We don't change the defaults than often so might not be a big maintenance issue.

Reply:
sorry, not maintained issue, but it will be issue or at least you have to dig a bit more into the documentation when developing, when using some api functions and have default values. So my point is to make it easy when doing api calls. @thc202

Reply:
Sure, was not arguing against adding the defaults.

Reply:
I think including the default details is a good move. When adding API endpoints it should be quick to memory anyway, and changes to defaults and VERY infrequent.